crates/starknet-types-core/benches/criterion_hashes.rs

```diff
@@ -0,0 +1,108 @@
+use criterion::{BatchSize, BenchmarkId, black_box, Criterion, criterion_group, criterion_main};
+use rand::{RngCore, SeedableRng};
+use rand_chacha::ChaCha8Rng;
+
+use starknet_types_core::felt::Felt;
+use starknet_types_core::hash::{Pedersen, Poseidon};
+use starknet_types_core::hash::StarkHash;
+
+const SEED: u64 = 3;
+
+fn rnd_felt(rng: &mut ChaCha8Rng) -> Felt {
+    let mut felt: [u8; 32] = Default::default();
+    rng.fill_bytes(&mut felt);
+    Felt::from_bytes_be(&felt)
```

Comment:
this will most likely only generate big numbers, we prob also want to add a random number `0 < x <= 32` to generate "more" random numbers

Reply:
Right!